.travis.yml

```diff
@@ -22,7 +22,7 @@ script:
   - mkdir build
```

Comment:
Changes in travis and appveyor are good.  Some questions pending about the CMakeLists changes though.  (See below).